grammars/html.cson

```diff
@@ -350,12 +350,16 @@
   'entities':
     'patterns': [
       {
-        'captures':
+        'begin': '(&)([a-zA-Z0-9]+|#[0-9]+|#x[0-9a-fA-F]+)'
+        'beginCaptures':
+          '1':
+            'name': 'punctuation.definition.entity.begin.html'
+          '2':
+            'name': 'entity.name.entity.other.html'
+        'end': '(;)'
```

Comment:
Hi @50Wliu and thanks for the suggestions. How do you think it should be? I don't know if I understand what you say.